functions/src/apis/users.ts

```diff
@@ -58,12 +58,13 @@ const getUsers = async (request: requestType, response: responseType) => {
       });
 };
 
+// should kill this route since it's redundant of triggerUserCreation
```

Comment:
Agree. We can add other pertinent user data with the patch route.

Reply:
I'll do some cleanup in a later PR.